src/scss/infoBlock.scss

```diff
@@ -0,0 +1,53 @@
+.info-block-wrapper {
+  font-size: 20px;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  z-index: 1000;
+  user-select: none;
+}
+
+.info-block {
+  position: relative;
+  top: 10%;
+  width: 600px;
+  height: auto;
+  margin: 0 auto;
+  padding: 10px;
+  border-radius: 20px;
+  border: 2px solid black;
+  background: linear-gradient(180deg, rgba(255,255,255,1) 0%, rgba(153,187,255,1) 100%);
+}
+
+.info-block-button {
+  position: absolute;
+  background-color: #f2f20d;
+  right: -2px;
+  bottom: -2px;
```

Comment:
tip: this code could be done using following construction:
```
.info-block {
  /* some code */
  &-button {
    /* some code */
  }
 &-wrapper {
  /* some code */
 }
}
```

Reply:
Done